invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/intro_section.html

```diff
@@ -54,11 +55,11 @@ <h2 class="section-title">
         <div class="intro-step-list-image">3</div>
       </div>
       <div class="content">
-        <div class="header">{{ _("Run it!") }}</div>
+        <div class="header">{{ _("Use it!") }}</div>
         <div class="description">
-          {{ _("Run your InvenioRDM instance locally or in containers.") }}
-          <a href="https://inveniordm.docs.cern.ch/install/build-setup-run/#run_1" target="_blank">
-            {{ _("Run") }}
+          {{ _("Use and explore your InvenioRDM instance.") }}
+          <a href="https://inveniordm.docs.cern.ch/use/" target="_blank">
```

Comment:
I was debating whether a deployment link would fit better here....but since that section is a bit underdeveloped "use" is probably fine.